server/Makefile-flash-att-v2

```diff
@@ -1,29 +1,18 @@
-flash_att_v2_commit_cuda := v2.5.8
+flash_att_v2_commit_cuda := v2.5.9.post1
 flash_att_v2_commit_rocm := 2554f490101742ccdc56620a938f847f61754be6
 
+build-flash-attention-v2-cuda:
+	pip install -U packaging wheel
```

Comment:
Wheel is necessary and `flash-attn` doesn't respect pep517 (https://github.com/Dao-AILab/flash-attention/issues/453) so nothing really better is doable I feel